python/feldera/rest/_httprequests.py

```diff
@@ -0,0 +1,150 @@
+
+    def send_request(
+            self,
+            http_method: Callable,
+            path: str,
+            body: Optional[
+                Union[Mapping[str, Any], Sequence[Mapping[str, Any]], List[str], str]
+            ] = None,
+            content_type: str = "application/json",
+            params: Optional[Mapping[str, Any]] = None,
+            stream: bool = False,
+    ) -> Any:
+        """
+        :param http_method: The HTTP method to use. Takes the equivalent `requests.*` module. (Example: `requests.get`)
+        :param path: The path to send the request to.
+        :param body: The HTTP request body.
+        :param content_type: The value for `Content-Type` HTTP header. "application/json" by default.
+        :param params: The query parameters part of this request.
+        """
+        self.headers["Content-Type"] = content_type
+
+        try:
+            timeout = self.config.timeout
+            headers = self.headers
+
+            request_path = self.config.url + "/" + self.config.version + path
+
+            logging.debug(
+                "sending %s request to: %s with headers: %s, and params: %s",
+                http_method.__name__, request_path, str(headers), str(params)
+            )
+
+            if http_method.__name__ == "get":
+                request = http_method(
+                    request_path,
+                    timeout=timeout,
+                    headers=headers,
+                    params=params,
+                )
+            elif isinstance(body, bytes):
+                request = http_method(
+                    request_path,
+                    timeout=timeout,
+                    headers=headers,
+                    data=body,
+                    params=params,
+                    stream=stream,
+                )
+            else:
+                request = http_method(
+                    request_path,
+                    timeout=timeout,
+                    headers=headers,
+                    data=json.dumps(body) if body else "" if body == "" else "null",
+                    params=params,
+                    stream=stream,
+                )
+                if stream:
+                    return request
+            resp = self.__validate(request)
+            logging.debug("got response: %s", str(resp))
+            return resp
+
+        except requests.exceptions.Timeout as err:
+            raise FelderaTimeoutError(str(err)) from err
+        except requests.exceptions.ConnectionError as err:
+            raise FelderaCommunicationError(str(err)) from err
+
+    def get(
+            self,
+            path: str,
+            params: Optional[Mapping[str, Any]] = None
+    ) -> Any:
+        return self.send_request(requests.get, path, params)
+
+    def post(
+            self,
+            path: str,
+            body: Optional[
+                Union[Mapping[str, Any], Sequence[Mapping[str, Any]], List[str], str]

```

Comment:
Well it can be `Optional[Any]`. 
I took this from: https://github.com/meilisearch/meilisearch-python/blob/00e7ce4ea450979eb7249796fbe412cc28fdabef/meilisearch/_httprequests.py#L30 and didn't change it.